tools/export/gcc_arm_nrf51_dk.tmpl

```diff
@@ -1,7 +1,11 @@
 {% extends "gcc_arm_common.tmpl" %}
 
+{% block target_all %}
+all: $(PROJECT).bin $(PROJECT).hex size merge
```

Comment:
If you follow my other comment and now have a rule for `$(PROJECT)-combined.hex`, Then this should be 
```make
all: $(PROJECT).bin $(PROJECT)-combined.hex size
```